src/components/Homepage/homepage.scss

```diff
@@ -90,106 +122,114 @@
 .homepage--callout {
   position: relative;
   z-index: 2;
-  padding-top: 2rem;
+  padding-top: 6rem;
   padding-bottom: 4rem;
   background: $carbon--gray-100;
 }
 
+.homepage--callout-header h3 {
+  @include carbon--type-style('expressive-heading-03', true);
+}
+
 aside h4 {
   margin-bottom: 2rem;
 }
 
-.homepage--callout--pink {
+// Pink Tiles
+
+.homepage--callout--tiles {
   position: relative;
   z-index: 2;
   padding-bottom: 4rem;
   background: $carbon--gray-100;
-  background: linear-gradient(to right, #ed558f, #6e0736);
   z-index: 2;
 }
 
-.homepage--callout a {
-  color: $carbon--blue-40;
-  text-decoration: none;
+.homepage--callout--tiles a {
+  color: $carbon--white-0;
 }
 
-.homepage--callout a:hover {
-  color: $carbon--blue-40;
-  text-decoration: underline;
+.homepage--tile-border {
+  outline: $carbon--magenta-50 solid 1px;
 }
 
-.homepage--footer__info {
-  display: flex;
-  flex-direction: column;
-  align-items: flex-start;
+.homepage--tile-border:hover {
+  background-color: $carbon--magenta-50;
+  transition: 150ms cubic-bezier(0.2, 0, 0.38, 0.9);
```

Comment:
Carbon has some pre-defined transitions you can use `transition: $transition--base $bx--standard-easing;` for example. Docs are on the Motion page near the bottom https://www.carbondesignsystem.com/guidelines/motion/basics#duration-tokens